pkg/operator/monitoring/metrics.go

```diff
@@ -0,0 +1,91 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package monitoring
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	appID = "app_id"
+)
+
+var (
+	serviceCreatedTotal = stats.Int64(
+		"operator/service_created_total",
+		"The total number of dapr services created.",
+		stats.UnitDimensionless)
+	serviceDeletedTotal = stats.Int64(
+		"operator/service_deleted_total",
+		"The total number of dapr services deleted.",
+		stats.UnitDimensionless)
+	serviceUpdatedTotal = stats.Int64(
+		"operator/service_updated_total",
+		"The total number of dapr services updated.",
+		stats.UnitDimensionless)
+
+	// appIDKey is a tag key for App ID
+	appIDKey = tag.MustNewKey(appID)
+)
+
+// RecordServiceCreatedCount records the number of dapr service created
+func RecordServiceCreatedCount(appID string) {
+	stats.RecordWithTags(context.Background(), withTags(appIDKey, appID), serviceCreatedTotal.M(1))
+}
+
+// RecordServiceDeletedCount records the number of dapr service deleted
+func RecordServiceDeletedCount(appID string) {
+	stats.RecordWithTags(context.Background(), withTags(appIDKey, appID), serviceDeletedTotal.M(1))
+}
+
+// RecordServiceUpdatedCount records the number of dapr service updated
+func RecordServiceUpdatedCount(appID string) {
+	stats.RecordWithTags(context.Background(), withTags(appIDKey, appID), serviceUpdatedTotal.M(1))
+}
+
+// withTags converts tag key and value pairs to tag.Mutator array.
```

Comment:
I think we need to move withTags util to common package :) we can do later.

Reply:
sure.